packages/issuer/lib/state-manager/IssuerStateBuilder.ts

```diff
@@ -0,0 +1,25 @@
+import { CredentialOfferV1_0_11, IssuerState } from '@sphereon/openid4vci-common'
+
+export class IssuerStateBuilder {
```

Comment:
a builder for an object which is not being created by a developers/lib user is a bit overkill for next time ;)

Reply:
Yep and the builder that I need to extend is in VDX-171.